frontend/src/components/panels/LayerTree.vue

```diff
@@ -293,9 +295,12 @@ const blendModeEntries: SectionsOfMenuListEntries<BlendMode> = [
 	],
 ];
 
-const RANGE_TO_INSERT_WITHIN_BOTTOM_FOLDER_NOT_ROOT = 40;
-const LAYER_LEFT_MARGIN_OFFSET = 28;
+const RANGE_TO_INSERT_WITHIN_BOTTOM_FOLDER_NOT_ROOT = 20;
 const LAYER_LEFT_INDENT_OFFSET = 16;
+const LAYER_LEFT_MARGIN_OFFSET = 4 + 32 + LAYER_LEFT_INDENT_OFFSET;
+const INSERT_MARK_OFFSET = 2;
```

Comment:
I couldn't figure out based on this name what this does. Could you please pick a more descriptive name? If it ends up requiring too many words, a name that's a summary would be good paired with a comment here.

Also a comment above `RANGE_TO_INSERT_WITHIN_BOTTOM_FOLDER_NOT_ROOT`, and possibly a rename if you can think of a more appropriate name, would be helpful since I'm still confused what that one does. (If I recall correctly, I renamed it some time ago but I may have had a mistaken understanding of what it did.) Thanks :)